lms/djangoapps/email_marketing/signals.py

```diff
@@ -90,15 +90,15 @@ def add_email_marketing_cookies(sender, response=None, user=None,
 
 
 @receiver(REGISTER_USER)
-def email_marketing_register_user(sender, user=None, profile=None,
+def email_marketing_register_user(sender, user=None, registration=None,
```

Comment:
What happens if the parameters `user` and `registration` are required? Does this fit with Celery architecture?